Gemfile.lock

```diff
@@ -264,11 +264,11 @@ GEM
       net-protocol
       timeout
     nio4r (2.5.8)
-    nokogiri (1.13.5-arm64-darwin)
```

Comment:
Think this is already in main if you rebase?

Reply:
> I think we also need to add the validation screen if you answer preg_occ first and then change a tenant gender or age question such that preg_occ needs confirming again?

@baarkerlounger from what I gather from the discussion in the ticket, might be worth deferring that part until a further discussion